sdk/dotnet/Provider.cs

```diff
@@ -278,8 +275,8 @@ public InputList<string> ImpersonateServiceAccountDelegates
         [Input("resourceManagerCustomEndpoint")]
         public Input<string>? ResourceManagerCustomEndpoint { get; set; }
 
-        [Input("resourceManagerV2beta1CustomEndpoint")]
-        public Input<string>? ResourceManagerV2beta1CustomEndpoint { get; set; }
+        [Input("resourceManagerV2CustomEndpoint")]
+        public Input<string>? ResourceManagerV2CustomEndpoint { get; set; }
```

Comment:
Also here

Reply:
The renaming of the custom endpoint is because they have moved to GA rather than using the beta endpoint

I am ok with the fact that this would affect a super small number of users and we can note this on the CHANGELOG as it's only a config time parameter